CHANGELOG.md

```diff
@@ -5,14 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- The `--outDir` flag is now required when using the command line tool. Previously it would print all concatenated typings to `stdout`, which doesn't make much sense given that we emit multiple files.
+
+## [1.0.0] - 2018-01-25
+- [BREAKING] The `--outDir` flag is now required when using the command line tool. Previously it would print all concatenated typings to `stdout`, which doesn't make much sense given that we emit multiple files.
 - Rewrite triple-slash references to Polymer into the `types/` directory so that they resolve correctly. Polymer is a special case where we put the typings in a `types/` subdirectory in order not to clutter the repo.
 - Emit a `const FooBehavior: object` for behaviors. This lets TypeScript know that e.g. `Polymer.AppLocalizeBehavior` is a valid symbol that could be passed, for example, to the `Polymer.mixinBehaviors` function.
 
-## [0.3.6] - 2017-01-09
+## [0.3.6] - 2018-01-09
 - Support parameterized types other than `Array` and `Object`, such as `Foo<T>`.
 
-## [0.3.5] - 2017-01-02
+## [0.3.5] - 2018-01-02
```

Comment:
Heh, good catch

Reply:
I have been doing this every year since grade school.